CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - [#1980](https://github.com/FuelLabs/fuel-core/pull/1980): Add `Transaction` to relayer 's event filter
 
 #### Breaking
+- [#2001](https://github.com/FuelLabs/fuel-core/pull/2001): Prevent GraphQL query body to be huge and cause OOM. The default body size is `1MB`. The limit can be changed by the `graphql-max-queries-body-limit` CLI argument.
```

Comment:
Can we reference `bytes` in this variable name? As I read it, it sounds like this value influences the max size of the graph query, not the total byte size of the request. Maybe something like `graphql-max-request-body-bytes`?